ryanair/ryanair.py

```diff
@@ -25,6 +25,16 @@
 logger.addHandler(console_handler)
 
 
+class RyanairException(Exception):
+    def __init__(self, message):
+        super().__init__(f"Ryanair API: {message}")
+
+
+class ShieldException(RyanairException):
+    def __init__(self):
+        super().__init__("Availibility declined, perhaps you are being throttled")
```

Comment:
Could we please update this to simply "Availablity API declined to provide a result", as it looks like this is going to be essentially normal for now, not caused by throttling.

I think it should also no longer be called `ShieldException`, but perhaps more simply `AvailabilityException` or `AvailabilityDecli edException`.

I'd also consider including the `response.json()` in the logged error, in case this includes more info in the future.

Reply:
I did modifications, however, calling response.json() somewhere here is not a good option IMHO